codebundles/curl-gmp-kong-ingress-inspection/runbook.robot

```diff
@@ -54,13 +54,17 @@ Check If Kong Ingress HTTP Request Latency Violates Threshold
     ...    target_service=${GCLOUD_SERVICE}
     ...    env=${env}
     ...    secret_file__gcp_credentials_json=${gcp_credentials_json}
+    ${svc_name}=    RW.CLI.Run Cli
```

Comment:
Why are we running a subsequent command on the shell instead of just parsing the output that we already have with RW.CLI.Parse CLI Output By Line? Was it just easier than the capture group regex pattern?

Reply:
I was following a previous examples here. Is there a better way, I am not intimately familiar with what we have available alternatively.

ie.
https://github.com/runwhen-contrib/rw-cli-codecollection/blob/main/codebundles/curl-gmp-kong-ingress-inspection/runbook.robot#L26
and 
https://github.com/runwhen-contrib/rw-cli-codecollection/blob/main/codebundles/curl-gmp-nginx-ingress-inspection/runbook.robot#L37

Reply:
@stewartshea for now this approach is simpler than capture groups, and I figured more intuitive since it's via grep+awk or any CLIs of your choosing. As we sus out next_steps if this proves insufficient I can always add in the capture groups.